Feature: add blog for index operation UI (#1365)
opensearch-project/project-website: _authors/lxuesong.markdown

---
name: Xuesong Luo
short_name: lxuesong
photo: '/assets/media/authors/lxuesong.jpeg'
github: xluo-aws
---

> **Xuesong Luo** is an Engineer Manager of Shanghai AWS OpenSearch Team

Reviewer: ```suggestion
**Xuesong Luo** is an AWS engineering manager working on the OpenSearch Project.
```

Reviewer: Should the other names be bold as well?

> **Xuesong Luo** is an AWS engineering manager working on the OpenSearch Project.

Author: Agree, done for this
